### PR TITLE
Logger PR review

### PR DESCRIPTION
**Pull request by BorjaOuterelo**
Fixes and modifications for https://github.com/eProsima/Micro-XRCE-DDS-Agent/pull/77.

### CODE REVIEW

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro%20XRCE-DDS%20Agent%20Manual%20Linux/185/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro%20XRCE-DDS%20Agent%20Manual%20Windows/188/)

File: src/cpp/Root.cpp [hunk 1]
```diff
@@ -86,9 +86,9 @@ dds::xrce::ResultStatus Root::create_client(
                         = std::make_shared<ProxyClient>(client_representation, middleware_kind);
                 if (clients_.emplace(client_key, std::move(new_client)).second)
                 {
-                    UXR_AGENT_LOG_INFO(
-                        UXR_CREATE_SESSION_PATTERN,
+                    UXR_AGENT_LOG_DEBUG(
```
[the remark this answers is not in the document]
Comment (BorjaOuterelo) [reply src/cpp/Root.cpp]: not really.

Comment (BorjaOuterelo) [inline src/cpp/Root.cpp]:
```suggestion
                    UXR_AGENT_LOG_INFO(
```

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro%20XRCE-DDS%20Agent%20Manual%20Linux/188/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro%20XRCE-DDS%20Agent%20Manual%20Windows/191/)

**Pull request merged**